ts/features/wallet/onboarding/paypal/screen/PayPalOnboardingCheckoutScreen.tsx

```diff
@@ -114,14 +116,29 @@ const PayPalOnboardingCheckoutScreen = (props: Props) => {
     navigation.dispatch(navigateToPayPalCheckoutCompleted());
   };
 
+  // notify the user that the current onboarding operation will be interrupted
+  const handleBack = () => {
+    Alert.alert(I18n.t("wallet.abortWebView.title"), "", [
+      {
+        text: I18n.t("wallet.abortWebView.confirm"),
+        onPress: props.goBack,
+        style: "cancel"
+      },
+      {
+        text: I18n.t("wallet.abortWebView.cancel")
+      }
+    ]);
+  };
+
   return (
     <BaseScreenComponent
-      goBack={props.goBack}
+      goBack={handleBack}
```

Comment:
Works well!
I suggest adding a test in [PayPalOnboardingCheckoutScreen.test.tsx](https://github.com/pagopa/io-app/blob/1c2d2b54b96ad5b075e702857f8517b1ea06759d/ts/features/wallet/onboarding/paypal/screen/__tests__/PayPalOnboardingCheckoutScreen.test.tsx#L17) asserting that once the user presses the back button, `Alert.alert` is called.

Reply:
Thank you! I should have included it.
Added https://github.com/pagopa/io-app/pull/3629/commits/732b86c81d67a751613a5aa56205c663aed54ecb